src/App/Page.php

```diff
@@ -267,9 +268,9 @@ private function initFooter(App $app, Mode $mode, L10n $l10n)
 		 */
 		if ($mode->isMobile() || $mode->isTablet()) {
 			if (isset($_SESSION['show-mobile']) && !$_SESSION['show-mobile']) {
-				$link = 'toggle_mobile?address=' . urlencode(curPageURL());
+				$link = 'toggle_mobile?address=' . urlencode(DI::baseUrl());
 			} else {
-				$link = 'toggle_mobile?off=1&address=' . urlencode(curPageURL());
+				$link = 'toggle_mobile?off=1&address=' . urlencode(DI::baseUrl());
```

Comment:
`DI::baseUrl()` doesn't have the current page URI, does it?

Reply:
It's a shortcut for `DI::baseUrl()->get()`, which says:

https://github.com/friendica/friendica/blob/c8a322baf0b95ea15bfa8031d77a555b2a13f0bf/src/App/BaseURL.php#L126-L135

Reply:
so yes, it does have

Reply:
I thought the base URL was the base URL of the Friendica node all this time?

Reply:
hm .. Currently there "should" be no difference, otherwise I'm not sure if there aren't sideeffects at other places too ..

Reply:
Guess who's combing the code now?

Reply:
Hm .. I see what you mean ..